### PR TITLE
fix: UAT nodo monitoring

### PR DESCRIPTION
**Pull request by aomegax**
### List of changes
- new product policy
- removed decoupler for monitoring

NDP monitoring : 

👀 if doesn't  work on u browser try Incognito modality or empty cache

- https://api.dev.platform.pagopa.it/nodo-monitoring/monitoring/v1/monitor 🆓
- https://api.uat.platform.pagopa.it/nodo-monitoring/monitoring/v1/monitor ⛔️ VPN PagoPA onprem or VPN Nexi 
- https://api.platform.pagopa.it/nodo-monitoring/monitoring/v1/monitor ➡️  _403 Forbidden, future needs_ ⛔️ VPN PagoPA onprem or VPN Nexi

```
sh terraform.sh apply <ENV> \
-target=module.apim_nodo_dei_pagamenti_monitoring_product \
-target=azurerm_api_management_api_version_set.nodo_monitoring_api \
-target=module.apim_nodo_monitoring_api
```

### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/core/apim_nodo_monitoring_services.tf [hunk 1]
```diff
@@ -16,15 +16,9 @@ module "apim_nodo_dei_pagamenti_monitoring_product" {
...
-    address-range-from       = var.env_short != "d" ? "10.1.128.0" : "0.0.0.0"
-    address-range-to         = var.env_short != "d" ? "10.1.128.255" : "0.0.0.0"
-    base-url                 = azurerm_api_management_named_value.default_nodo_backend.value
-    base-node-id             = azurerm_api_management_named_value.default_nodo_id.value
-    is-nodo-auth-pwd-replace = false
-    }) : templatefile("./api_product/nodo_pagamenti_api/_base_policy.xml", { # decoupler OFF
-    address-range-from = var.env_short != "d" ? "10.1.128.0" : "0.0.0.0"
-    address-range-to   = var.env_short != "d" ? "10.1.128.255" : "0.0.0.0"
+  policy_xml = templatefile("./api_product/nodo_pagamenti_api/monitoring/base_policy.xml.tpl", {
+    base-url = var.env_short == "p" ? "https://{{ip-nodo}}" : "http://{{aks-lb-nexi}}{{base-path-nodo-oncloud}}"
```
Comment (aomegax) [inline src/core/apim_nodo_monitoring_services.tf:20]: do you evaluate to adopt `namedValue` as `azurerm_api_management_named_value.default_nodo_backend.value`?

File: src/core/apim_nodo_monitoring_services.tf [hunk 2]
```diff
@@ -75,7 +69,10 @@ module "apim_nodo_monitoring_api" {
   })
 
   xml_content = templatefile("./api/nodopagamenti_api/monitoring/v1/_base_policy.xml.tpl", {
-    base-url                  = azurerm_api_management_named_value.default_nodo_backend.value
-    is-nodo-decoupler-enabled = var.apim_nodo_decoupler_enable
+    base-url     = var.env_short == "p" ? "https://{{ip-nodo}}" : "http://{{aks-lb-nexi}}{{base-path-nodo-oncloud}}"
```
Comment (aomegax) [inline src/core/apim_nodo_monitoring_services.tf:72]: the logic adopted in the node policies is to always refer to the `default-node-backend` namedValue

https://github.com/pagopa/pagopa-infra/blob/main/src/core/apim.tf#L329

**Review by aomegax — COMMENTED** [review]
my 2 cents about policies uniformity

**Review by andreapasuch — DISMISSED** [review]
ok

File: src/core/apim_nodo_monitoring_services.tf (hunk 1 above)
Comment (pasqualespica) [reply src/core/apim_nodo_monitoring_services.tf:20]: ok

**Pull request merged**